test/com/linkedin/drelephant/util/InfoExtractorTest.java

```diff
@@ -218,14 +218,14 @@ public boolean isEmpty() {
     InfoExtractor.loadSchedulerInfo(result, data, scheduler);
 
     assertEquals(result.scheduler, "azkaban");
-    assertFalse(StringUtils.isEmpty(result.getJobExecId()));
-    assertFalse(StringUtils.isEmpty(result.getJobDefId()));
-    assertFalse(StringUtils.isEmpty(result.getFlowExecId()));
-    assertFalse(StringUtils.isEmpty(result.getFlowDefId()));
-    assertFalse(StringUtils.isEmpty(result.getJobExecUrl()));
-    assertFalse(StringUtils.isEmpty(result.getJobDefUrl()));
-    assertFalse(StringUtils.isEmpty(result.getFlowExecUrl()));
-    assertFalse(StringUtils.isEmpty(result.getFlowDefUrl()));
+//    assertFalse(StringUtils.isEmpty(result.getJobExecId()));
```

Comment:
Bah, sorry, I missed that. I think I got compilation errors: Cannot resolve method...

Reverted.